src/consensus/aft/request.h

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+
+#include "ds/json.h"
+#include "kv/map.h"
+#include "node/entities.h"
+
+#include <msgpack/msgpack.hpp>
+#include <vector>
+
+namespace aft
+{
+  struct Request
+  {
+    uint64_t caller_id;
+    std::vector<uint8_t> caller_cert;
+    std::vector<uint8_t> raw;
+    std::vector<uint8_t> pbft_raw;
```

Comment:
Dedupe

Reply:
This turns out to be a very large change. I am going to do this in a separate PR.